### PR TITLE
the intern skills percentage showing as two decimal places now.

### PR DESCRIPTION
**Pull request by frzana1012**

### CODE REVIEW

File: main/UseCases/GameGenerators.java [hunk 1]
```diff
@@ -118,15 +120,19 @@ public void generateFinalProject() throws FileNotFoundException {
      * @param skillList an ArrayList of skills.
      * @return a HashMap of Skills and the score (0.00 - 1.00) of the skill
      */
-    private HashMap<String, Double> generateUniqueSkillMap(ArrayList<String> skillList){
```
Comment (Jacobcansleepnow) [inline main/UseCases/GameGenerators.java:121]: Should keep as private, please do change back to private.

Comment (frzana1012) [reply main/UseCases/GameGenerators.java:121]: sure!

File: main/UseCases/GameGenerators.java [hunk 2]
```diff
@@ -118,15 +120,19 @@ public void generateFinalProject() throws FileNotFoundException {
...
+
+    public HashMap<String, Double> generateUniqueSkillMap(ArrayList<String> skillList){
         Random random = new Random();
         HashMap<String, Double> skillMap = new HashMap<>();
+        double d = 100.00 * random.nextDouble();
```
Comment (Jacobcansleepnow) [inline main/UseCases/GameGenerators.java:127]: should declare inside for loop!

**Review by Jacobcansleepnow — CHANGES_REQUESTED** [review]
Commented in code

**Review by Jacobcansleepnow — APPROVED** [review]
Fixed!

**Pull request merged**